components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -333,6 +333,12 @@ func (gp *APIoverJSONRPC) handler(ctx context.Context, conn *jsonrpc2.Conn, req
 		default:
 		}
 	}
+	for chn := range gp.subs[""] {
```

Comment:
not sure what it is for

Reply:
This allows subscriptions to arbitrary instance ID's, not just a specific one.